pyproject.toml

```diff
@@ -173,7 +174,7 @@ visualization = ["graphviz"]
 decompiler = ["black"]
 deltalake = ["deltalake"]
 examples = ["pins", "fsspec"]
-geospatial = ["geopandas", "shapely"]
+geospatial = ["geoarrow-types", "geopandas", "shapely"]
```

Comment:
Since we're importing pyproj directly, we need to add it as an optional dependency for the geospatial extra, I can take care of that.

Reply:
Done!